ArjunaJTA/object_store/run-jgroups-cluster.sh

```
@@ -0,0 +1,63 @@
+    TIMEOUT=$((TIMEOUT - 1))
+done
+
+if [ ! -f node1.ready ]; then
+    echo "FAILED: node1 did not become ready within 30 seconds"
+    kill $PID1 2>/dev/null
+    exit 1
+fi
+
+echo "Starting node2 on port 7801..."
+mvn -q exec:java -Dexec.mainClass=$MAIN_CLASS  -q exec:java -Dexec.mainClass=$MAIN_CLASS \
+    -Dexec.args="node2" -Djgroups.bind_port=7801 $NARAYANA_VERSION_PARAM
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)